### PR TITLE
balloon_stress: fix syntax error

### PR DESCRIPTION
**Pull request by PandaWei**
fix omission of 82f52f8

Signed-off-by: Wei Jiangang <weijg.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by PandaWei** [comment]
@CongLi  I check #626 again and found a omission.

File: qemu/tests/balloon_stress.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def run(test, params, env):
     session.cmd(params["stop_player_cmd"])
     vm.verify_alive()
 
-    if (params.get("need_clear_verifier"), "yes") == "yes":
+    if (params.get("need_clear_verifier", "yes") == "yes":
```
Comment (CongLi) [inline qemu/tests/balloon_stress.py]: there is one more "(" here.

Comment (PandaWei) [inline qemu/tests/balloon_stress.py]: Updatre and remove the first '('

**Comment by CongLi** [comment]
Thanks @PandaWei.

Acked-by: Cong Li <coli@redhat.com>

File: qemu/tests/balloon_stress.py (hunk 1 above)
Comment (CongLi) [inline qemu/tests/balloon_stress.py]: Maybe your push was not successful. :(

Comment (PandaWei) [inline qemu/tests/balloon_stress.py]: @CongLi 
Yes, Push failed just now.
update

**Comment by CongLi** [comment]
Thanks @PandaWei 

Acked-by: Cong Li <coli@redhat.com>

**Pull request merged**